lib/usd/translators/jointWriter.cpp

```diff
@@ -198,14 +193,15 @@ static bool _FindDagPoseMembers(
             auto          it = pathIndexMap.find(connNode);
             if (it != pathIndexMap.end()) {
                 (*indices)[it->second] = i;
+                visitedIndices[it->second] = 1;
             }
         }
     }
 
     // Validate that all of the input dagPaths are members.
-    for (size_t i = 0; i < indices->size(); ++i) {
-        int index = (*indices)[i];
-        if (index < 0) {
+    for (size_t i = 0; i < visitedIndices.size(); ++i) {
+        int index = visitedIndices[i];
+        if (index != 1) {
```

Comment:
We still need to pull a value from `indices` here. The `visitedIndices` value is going to be either `0` or `1`:

```C++
    for (size_t i = 0u; i < visitedIndices.size(); ++i) {
        const bool visited = visitedIndices[i];
        if (!visited) {
            ... // Error
        }
        const unsigned int index = (*indices)[i];
        ...
```

Or if you prefer to use `uint8_t`:
```C++
    for (size_t i = 0u; i < visitedIndices.size(); ++i) {
        const uint8_t visited = visitedIndices[i];
        if (visited != 1u) {
            ... // Error
        }
        const unsigned int index = (*indices)[i];
        ...
```

Reply:
You're right; this is a silly mistake on my part, I totally missed that the warning string would have the wrong index otherwise. I have fixed this internally and @dgovil will submit the update soon. Thanks for catching this!

Reply:
Pushed the changes.

Reply:
Cool, no problem! With the truncated diff GitHub was showing, I failed to notice that we actually need the `index` value **in** the error message, so apologies for the slightly misleading suggestions. :) Sounds like you've got it covered though. Thanks!